### DISCUSSION TITLE
designable源代码的阅读有没有什么技巧

### DISCUSSION THREAD

**Discussion by jinphic** [alibaba/designable#106] [General]
这边想系统学习你们做这个项目的过程，从头开始阅读代码还是成本太大，有没有技巧或者图谱，希望得到经验和分享

**Comment by lidianhao123** [comment]
[Designable 应用和源码浅析](https://zhuanlan.zhihu.com/p/431263711) 试试我这篇文章是否对你有帮助

**Reply by jinphic** [reply]
嗯嗯，这个之前看到作者点赞你的文章了，谢谢分享
